.github/workflows/release-tag.yml

```
@@ -0,0 +1,37 @@
+              if: env.RELEASE_VERSION == ''
+              run: |
+                  echo "RELEASE_VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+
+            - name: Generate Release Notes
+              run: |
+                  ./.github/workflows/release-notes.js ${{ env.RELEASE_VERSION }}
+                  cat notes-${{ env.RELEASE_VERSION }}.md
+
+            - name: Create Release for Tag
+              id: release_tag
+              uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)